app/views/layouts/application.html.erb

```diff
@@ -1,7 +1,7 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>The Pivot</title>
+    <title>The Gallery</title>
```

Comment:
I think maybe we should keep the name of the business here, if someone is checking out our Heroku, they'll have no idea what the Pivot is.